.travis.yml

```diff
@@ -28,9 +32,13 @@ matrix:
         directories:
           - $HOME/Library/Caches/go-build
           - $HOME/gopath/pkg/mod
+    - os: osx
+      go: tip
+      cache:
+        directories:
+          - $HOME/Library/Caches/go-build
+          - $HOME/gopath/pkg/mod
 
-install:
-  - go get -v github.com/golangci/golangci-lint/cmd/golangci-lint
 script:
-  - go build ./...
-  - ./run_tests.sh
+  - env GO111MODULE=on GORACE=halt_on_error=1 go build -race ./...
```

Comment:
we intentionally removed -race before (in 15967e4b7989c723e93e58c9951188a8c39b6905)